test/matchtests.jl

```diff
@@ -209,36 +209,71 @@ end
 
 @assert @match([1 2 3 4; 5 6 7 8], [a b... c])              == ([1,5] , [2 3; 6 7] , [4,8])
 
-# match / collect rows
-@assert @match([1 2 3; 4 5 6], [a, b])                      == ([1 2 3], [4 5 6])
-@assert @match([1 2 3; 4 5 6], [[1 2 3], a])                ==  [4 5 6]             # TODO: don't match this
-@assert @match([1 2 3; 4 5 6], [1 2 3; a])                  ==  [4 5 6]
+if VERSION < v"0.5.0-dev"
```

Comment:
it's usually better to be more specific about VERSION cutoffs like this if possible, in case this package triggers a base bug that needs bisecting